### PR TITLE
show drop down for all round application in to approve tab

### PR DESCRIPTION
**Pull request by brajesh-lab**
Targets #{2509}

### Description
show drop down for all round application in to approve tab
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/views/hr/application/edit.blade.php [hunk 1]
```diff
@@ -30,8 +30,8 @@
                                     aria-controls="nav-pre-trial" aria-selected="false">Pre-Trial</a>
                             </div>
                         </nav>
-                        <div class="tab-content" id="nav-tabContent">
-                            <div class="tab-pane fade {{ $application->latestApplicationRound->round->isTrialRound() ? 'show active' : '' }}"
+                        <div  id="nav-tabContent">
```
Comment (P4NK4J) [inline resources/views/hr/application/edit.blade.php:33]: What does these additional classes do ?, I didn't see a bootstrap class with name nav-tabContent ?

File: resources/views/hr/application/edit.blade.php [hunk 2]
```diff
@@ -699,7 +699,7 @@ class="py-1 mb-0 btn btn-info btn-sm round-submit update-schedule"
                                                     </div>
                                                     <div class="form-row d-flex justify-content-end">
                                                         <button type="button" class="btn btn-info btn-sm round-submit"
-                                                            data-action="update">Update feedback</button>
+                                                            data-action="update">Update feedback702</button>
```
Comment (P4NK4J) [inline resources/views/hr/application/edit.blade.php:702]: what's feedback 702 ?

File: resources/views/hr/application/edit.blade.php [hunk 3]
```diff
@@ -1220,7 +1220,7 @@ class="py-1 mb-0 btn btn-info btn-sm round-submit update-schedule"
                                             @php
                                                 $showFooter = false;
                                                 if ($loop->last) {
-                                                    if (in_array($applicationRound->application->status, [config('constants.hr.status.sent-for-approval.label'), config('constants.hr.status.approved.label')])) {
+                                                    if (in_array($applicationRound->application->status, [config('constants.hr.status.sent-for-approval.label'), config('constants.hr.status.approved.label'),config('constants.hr.status.in-progress.label')])) {
```
Comment (P4NK4J) [inline resources/views/hr/application/edit.blade.php:1223]: what's the purpose of adding in progress label here ?

File: resources/views/hr/application/edit.blade.php [hunk 4]
```diff
@@ -1232,7 +1232,7 @@ class="py-1 mb-0 btn btn-info btn-sm round-submit update-schedule"
                                             @if ($showFooter)
                                                 <div class="card-footer">
                                                     <div class="d-flex align-items-center">
-                                                        @if ($applicationRound->showActions)
```
Comment (P4NK4J) [inline resources/views/hr/application/edit.blade.php:1235]: why this condition is removed ?

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left some comments, please check

File: resources/views/hr/application/edit.blade.php (hunk 3 above)
Comment (brajesh-lab) [reply resources/views/hr/application/edit.blade.php:1223]: To show footer for if application status is in progress label

File: resources/views/hr/application/edit.blade.php (hunk 1 above)
Comment (brajesh-lab) [reply resources/views/hr/application/edit.blade.php:33]: these classes hide the div  when clicking on any part of the these div tag

File: resources/views/hr/application/edit.blade.php (hunk 2 above)
Comment (brajesh-lab) [reply resources/views/hr/application/edit.blade.php:702]: sorry i will correct that it is made by mistake

File: resources/views/hr/application/edit.blade.php (hunk 4 above)
Comment (brajesh-lab) [reply resources/views/hr/application/edit.blade.php:1235]: to show drop down for all round of applications

**Review by P4NK4J — CHANGES_REQUESTED** [review]
@brajesh-lab If an application is evaluated in previous round, the take action bar should not appear there. Please fix that, and else looks good.
image

**Pull request closed without merging**